deploy/buildtools/bitbuilder.py

```diff
@@ -93,10 +93,10 @@ def __init__(self, build_config: BuildConfig, args: Dict[str, Any]) -> None:
     def _parse_args(self) -> None:
         """Parse bitbuilder arguments."""
         self.s3_bucketname = self.args["s3_bucket_name"]
-        if self.args["append_userid_region"]:
-            self.s3_bucketname += "-" + get_aws_userid() + "-" + get_aws_region()
-
         if valid_aws_configure_creds():
+            if self.args["append_userid_region"]:
```

Comment:
F1 recipes were blowing up everything on my azure instance, i'm not sure this change should remain, but we shouldn't query aws-specific things unless we know we can / are actually going to build the recipe.